From afcb2f81a0b928244b9ea9cfc92143018bd56226 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Wed, 2 May 2018 14:41:47 -0700
Subject: [PATCH 1/2] fix include libev

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 Makefile.am | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 3c0d16a..dc5a19b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -55,3 +55,6 @@ EXTRA_DIST = \
 	CHANGELOG \
 	LICENSE \
 	README.md
+
+i3lock_CFLAGS += \
+	-I/usr/include/libev
-- 
2.19.1

